torch/_inductor/codegen/cpp.py

```diff
@@ -2083,7 +2083,9 @@ def _used_by_to(to_node: torch.fx.Node):
                                 node.replace_all_uses_with(val_node)
                                 sub_graph.erase_node(node)
 
-                    sub_graph.lint()
+                    # throws bound method LoopBody.get_index of <torch._inductor.ir.LoopBody object
```

Comment:
cc @EikanWang would you take a look at this ? lint() throws because `LoopBody.get_index` is the `call_module` node but `LoopBody.get_index` is not an nn.Module

Reply:
Oh, sure. Let's check it.

Reply:
@eellison , may I know if this issue only happens under debug mode?

Reply:
For debug mode, the graph of `LoopBody` will attach a new `GraphModule` as `owning_module` while the release mode will not - https://github.com/pytorch/pytorch/blob/master/torch/_inductor/ir.py#L4203. The `lint` will check whether the graph has `owning_module` to decide if it needs to check `call_module` - https://github.com/pytorch/pytorch/blob/master/torch/fx/graph.py#L1338. Hence, we could patch the `owning_graph`  of `sub_graph` to `None` to move the `lint` onward. Eventually, I think we should call `get_index` via `call_function` but not `call_module`. I will submit a PR to fix this issue.